CHANGELOG.md

```diff
@@ -15,20 +15,20 @@
 
 #### Rename FreeRTOS-IoT-Libraries Directory
 
-- The FreeRTOS-IoT-Libraries directories in the previous snapshot version (V2020218_LTS_development_snapshot) under FreeRTOS-Plus/Source and FreeRTOS-Plus/Demo has been renamed to *FreeRTOS-IoT-Libraries-Beta1* to make way for a parallel directory, *FreeRTOS-IoT-Libraries-Beta2*, that holds libraries for which the refactoring work is near completion (currently the MQTT library).
+- The FreeRTOS-IoT-Libraries directories in the previous snapshot version (V2020218_LTS_development_snapshot) under `FreeRTOS-Plus/Source and FreeRTOS-Plus/Demo` has been renamed to *FreeRTOS-IoT-Libraries-Beta1* to make way for a parallel directory, *FreeRTOS-IoT-Libraries-Beta2*, that holds libraries for which the refactoring work is near completion (currently the MQTT library).
```

Comment:
It will be better if we break down this complex sentence. By the time I reach "(currently the MQTT library)" I don't know the context.  ( I know, but I think customer may not. )

Reply:
Grammar: _"The FreeRTOS-IoT-Libraries directories... __have__ been renamed..."_

Reply:
How about 
"to make way for a parallel directory, *FreeRTOS-IoT-Libraries-Beta2*, that holds libraries for which refactoring work is near completion. _Currently, the FreeRTOS-IoT-Libraries-Beta2 directories only contain the refactored MQTT library._"